src/Config/ConfigurationLoader.php

```diff
@@ -17,17 +17,18 @@
 
 class ConfigurationLoader implements IFlags, IBandits
 {
-    private const BANDIT_TIMESTAMP = "banditTimestamp";
-    private const BANDIT_MODEL_VERSIONS_KEY = 'banditModelVersions';
+    private const KEY_BANDIT_TIMESTAMP = "banditTimestamp";
+    private const KEY_LOADED_BANDIT_VERSIONS = 'banditModelVersions';
     private UFCParser $parser;
 
-    private const FLAG_TIMESTAMP = "flagTimestamp";
-    private const FLAG_ETAG = "flagETag";
+    private const KEY_FLAG_TIMESTAMP = "flagTimestamp";
+    private const KEY_FLAG_ETAG = "flagETag";
 
     public function __construct(
         private readonly APIRequestWrapper $apiRequestWrapper,
         private readonly IConfigurationStore $configurationStore,
-        private readonly int $cacheAgeLimit = 30
+        private readonly int $cacheAgeLimit = 30,
+        private readonly bool $optimizedBanditLoading = false
```

Comment:
🤔 Do we want to be able to turn optimized loading on and off? I can't think of a reason why we would want to make it user-configurable.
Added the parameter here in case we're not confident in model-matching or wanted to otherwise surface this option higher up (whether by env/ClientConfig). Will remove if it's not needed (which, seems like it's not).

Reply:
Agreed, think we can remove it

Reply:
Also agree we should remove it